README.md

```diff
@@ -82,6 +82,13 @@
     * [7.7.2. Fast-Aggregations](#772-fast-aggregations)
   * [7.8. Performance](#78-performance)
     * [7.8.1. Partition Aware](#781-partition-aware)
+    * [7.8.2. Near Cache](#782-near-cache)
+      * [7.8.2.1. Configuring Near Cache](#7821-configuring-near-cache)
+      * [7.8.2.2. Near Cache Example for Map](7822-near-cache-example-for-map)
```

Comment:
7822-near-cache-example-for-map -> #7822-near-cache-example-for-map

# is also missing for the other near cache items.